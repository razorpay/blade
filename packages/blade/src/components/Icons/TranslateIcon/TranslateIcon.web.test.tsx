import TranslateIcon from './';
import renderWithTheme from '~utils/testing/renderWithTheme.web';

describe('<TranslateIcon />', () => {
  it('should render TranslateIcon', () => {
    const { container } = renderWithTheme(
      <TranslateIcon color="feedback.icon.neutral.intense" size="large" />,
    );
    expect(container).toMatchSnapshot();
  });
});
