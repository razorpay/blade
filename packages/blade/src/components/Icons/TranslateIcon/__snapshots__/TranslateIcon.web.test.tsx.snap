// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TranslateIcon /> should render TranslateIcon 1`] = `
<div>
  <svg
    aria-hidden="true"
    class="Svgweb__StyledSvg-vcmjs8-0"
    data-blade-component="icon"
    fill="none"
    height="20px"
    viewBox="0 0 24 24"
    width="20px"
  >
    <path
      d="M6.19201 10.3022C7.87772 10.3022 8.93486 9.40506 8.93486 7.86709C8.93486 7.72469 8.92058 7.59653 8.9063 7.46838C9.07772 7.48262 9.26344 7.49684 9.43486 7.49684C9.66491 7.49684 9.91597 7.47589 10.1586 7.4389C10.3162 7.41487 10.4634 7.53349 10.4634 7.69289V10C10.4634 10.5523 10.9112 11 11.4634 11C12.0157 11 12.4634 10.5523 12.4634 10V4.14H12.7646C13.3163 4.14 13.7634 3.69282 13.7634 3.1412C13.7634 2.58958 13.3163 2.1424 12.7646 2.1424H10.5622C10.0106 2.1424 9.56344 2.58958 9.56344 3.1412V3.24C9.56344 3.73706 9.96638 4.14 10.4634 4.14V5.40579C10.4634 5.62465 10.3216 5.82093 10.1076 5.86666C9.75707 5.94154 9.39278 5.9731 9.02058 5.9731C8.63486 5.9731 8.29201 5.94462 7.97772 5.9019C8.46344 5.47468 8.72058 4.90506 8.72058 4.23576C8.72058 2.96835 7.96344 2 6.13486 2C5.44316 2 4.8186 2.119 4.2558 2.32474C3.89073 2.4582 3.73308 2.8712 3.86462 3.23696C4.02792 3.69106 4.56216 3.87653 5.02647 3.74505C5.31498 3.66335 5.60349 3.62342 5.89201 3.62342C6.44915 3.62342 6.79201 3.86551 6.79201 4.33544C6.79201 4.76147 6.53523 5.04627 5.708 5.2246C5.28043 5.31677 4.97426 5.71379 5.05385 6.14388C5.12413 6.52371 5.47196 6.7937 5.85437 6.73917C6.098 6.70444 6.32959 6.66006 6.54915 6.59968C6.86344 6.91298 7.04915 7.26899 7.04915 7.66772C7.04915 8.32278 6.63486 8.66456 5.94915 8.66456C5.0416 8.66456 4.36569 7.91939 3.65928 6.32093C3.45737 5.86405 2.92015 5.64934 2.47145 5.86884C2.07196 6.06426 1.88785 6.53685 2.07115 6.94205C3.21195 9.464 4.44972 10.3022 6.19201 10.3022Z"
      data-blade-component="svg-path"
      fill="hsla(211, 33%, 21%, 1)"
    />
    <path
      d="M16.7634 4.25H17.7634C19.4203 4.25 20.7634 5.59315 20.7634 7.25V12.95C20.7634 13.5023 21.2112 13.95 21.7634 13.95C22.3157 13.95 22.7634 13.5023 22.7634 12.95V7.25C22.7634 4.48858 20.5249 2.25 17.7634 2.25H16.7634C16.2112 2.25 15.7634 2.69772 15.7634 3.25C15.7634 3.80228 16.2112 4.25 16.7634 4.25Z"
      data-blade-component="svg-path"
      fill="hsla(211, 33%, 21%, 1)"
    />
    <path
      d="M7.76344 19.25C8.31572 19.25 8.76344 19.6977 8.76344 20.25C8.76344 20.8023 8.31572 21.25 7.76344 21.25C5.00201 21.25 2.76344 19.0114 2.76344 16.25V13.25C2.76344 12.6977 3.21115 12.25 3.76344 12.25C4.31572 12.25 4.76344 12.6977 4.76344 13.25V16.25C4.76344 17.9069 6.10658 19.25 7.76344 19.25Z"
      data-blade-component="svg-path"
      fill="hsla(211, 33%, 21%, 1)"
    />
    <path
      clip-rule="evenodd"
      d="M16.2634 10.25C16.6422 10.25 16.9884 10.464 17.1578 10.8028L21.6578 19.8028C21.9048 20.2968 21.7046 20.8974 21.2106 21.1444C20.7166 21.3914 20.116 21.1912 19.869 20.6972L18.6454 18.25H13.8814L12.6578 20.6972C12.4108 21.1912 11.8102 21.3914 11.3162 21.1444C10.8222 20.8974 10.622 20.2968 10.869 19.8028L15.369 10.8028C15.5384 10.464 15.8846 10.25 16.2634 10.25ZM16.2634 13.4861L17.6454 16.25H14.8814L16.2634 13.4861Z"
      data-blade-component="svg-path"
      fill="hsla(211, 33%, 21%, 1)"
      fill-rule="evenodd"
    />
  </svg>
</div>
`;
